scripts/rerun.py

```diff
@@ -0,0 +1,156 @@
+"""
+This script/module may be used to re-run, commit & push notebooks
+from the CLI or from another Python script (via `import rerun`).
+
+Run `python rerun.py -h` to show the CLI help.
+
+The example below does the following:
+1. Re-runs the BEST notebook
+2. Commits changes to a branch "rerun-best"
+3. Push that branch to a remote named "mine"
+   Assuming you did something like: `git add remote mine https://github.com/yourgithubusername/pymc-examples`
+
+```
+python scripts/rerun.py --fp_notebook=examples/case_studies/BEST.ipynb --commit_to=rerun-best --push_to=mine
+```
+"""
+import argparse
+import logging
+import pathlib
+import subprocess
+
+
+logging.basicConfig(level=logging.INFO)
+_log = logging.getLogger(__file__)
+DP_REPO = pathlib.Path(__file__).absolute().parent.parent
+
+REPLACEMENTS = {
```

Comment:
The order matters here.

Reply:
Ah, so that's why there's the `stundenat`→`studentt`!
I though you were correcting typos there..

Reply:
Should we not make this an OrderedDict or can we rely on `dict` being ordered now?